modules/layers/src/column-layer/column-layer.ts

```diff
@@ -245,18 +245,16 @@ export default class ColumnLayer<DataT = any, ExtraPropsT extends {} = {}> exten
 
   getShaders() {
     const {device} = this.context;
-    const transpileToGLSL100 = device.info.type !== 'webgl2';
     const defines: Record<string, any> = {};
 
-    const useDerivatives = this.props.flatShading && device.features.has('glsl-derivatives');
+    const useDerivatives = this.props.flatShading;
```

Comment:
Do we still want to call it `useDerivatives`? That seems to be "looking inside the black box", i.e. flatShading happens to be implemented with derivatives.